Added programmatic ability to add publish layout tabs for specific channels

asadmunir opened the pull request:
The function argument was already there but never used.

<!--

What's in this pull request?

The title of the pull request should look like the change log line, with reference to the corresponding issue number if available. For example:
  - Resolved #1234 where <something> was causing template parsing error 
(or)
  - Added ability to <do something new>; #1234

In the pull request body, give a good description of what the nature of the change is, and the reasoning behind it. Provide links to external references/discussions if appropriate.

If this pull request resolves a project issue, provide a link with a closing keyword. For example:
  Resolves https://github.com/ExpressionEngine/ExpressionEngine/issues/1235

Assign this PR the appropriate label depending on what it does, e.g. 'Bug:Accepted', or 'enhancement'

If documentation update is needed, provide a link to the corresponding pull request in the ExpressionEngine-User-Guide repo. For example:
  User Guide Pull Request: https://github.com/ExpressionEngine/ExpressionEngine-User-Guide/pull/1235

-->

asadmunir committed:
fix: add ability to add layout tabs channels specific

The function argument was already there but never used.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/ExpressionEngine/ExpressionEngine?pullRequest=3737) <br/>All committers have signed the CLA.

intoeetive reviewed (changes requested):
@asadmunir thank you for this addition!

Can you please just make some changes in code styling: we're not adding spaces inside brackets (so `array($channel_id)` and not `array( $channel_id )` etc, and also let the second `->filter(` go on its own line

Thank you!

asadmunir committed:
fix: matches code format

My code format was different so I changed my code format to match the EE code format.

asadmunir commented:
Hey Yuri, thank you for the instructions, I have fixed the code
formatting issue and committed my change. You can review my PR now. Thank
you.


On Thu, Aug 31, 2023 at 3:40 PM Yuri Salimovskiy ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
>
> @asadmunir <https://github.com/asadmunir> thank you for this addition!
>
> Can you please just make some changes in code styling: we're not adding
> spaces inside brackets (so array($channel_id) and not array( $channel_id )
> etc, and also let the second ->filter( go on its own line
>
> Thank you!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ExpressionEngine/ExpressionEngine/pull/3737#pullrequestreview-1604465965>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABHFNYGC3IJLZNTYB3HPYNLXYBS3JANCNFSM6AAAAAA4FYAAPM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

intoeetive reviewed (approved):
great!
